python/lsst/summit/utils/nightReport.py

```diff
@@ -636,7 +638,8 @@ def plotPerObjectAirMass(
             obsTimes = [self.getExposureMidpoint(seqNum) for seqNum in seqNums]
             color = self.cMap[star].color
             marker = self.cMap[star].marker
-            plt.plot(obsTimes, airMasses, color=color, marker=marker, label=star, ms=10, ls="")
+            obsDates = date2num(obsTimes)
+            plt.plot(obsDates, airMasses, color=color, marker=marker, label=star, ms=10, ls="")
```

Comment:
Argument 1 to "plot" has incompatible type "list[datetime]"; expected "float | _SupportsArray[dtype[Any]] | _NestedSequence[_SupportsArray[dtype[Any]]] | bool | int | float | complex | str | bytes | _NestedSequence[bool | int | float | complex | str | bytes] | str"

Reply:
Grrr, that's a matplotlib bug then, I guess, because it actually works just fine, and it's obviously doing something clever with them, and it should declare that I think. This change is fine though, thanks for that.